bugbug/model.py

```diff
@@ -402,12 +409,15 @@ def train(self, importance_cutoff=0.15):
         # Evaluate results on the test set for some confidence thresholds.
         for confidence_threshold in [0.6, 0.7, 0.8, 0.9]:
             y_pred_probas = self.clf.predict_proba(X_test)
+            confidence_class_names = self.class_names + ["Not classified"]
```

Comment:
Use "___NOT_CLASSIFIED___" (so it's clear that it is just an internal name). Then when you print it you use "Not classified"